i18n/locales/active.en.json

```diff
@@ -10,17 +10,18 @@
   "ElapsedModeDescription": "Will show the elapsed time in Discord",
   "EnableDevicesByDefault": "Enable new devices automatically",
   "EnableDevicesByDefaultDescription": "Should new discovered devices be enabled by default",
+  "MovieDetails": "Movie details on Plex",
```

Comment:
For the translations, please still run all the commands (even if not modifying the translations), it will prevent the program to panic if an entry in a language (French, randomly) does not exist :)

Reply:
Oh I wasn't even aware of the goi18n CLI oops I just added those manually lol. Should've probably read the entire readme. Will do!

Reply:
I've added the translate.fr.json file for you to edit

Reply:
Oh I suppose you wanted me to add the English strings in active.fr.json so I'll just do that instead

Reply:
I've added the untranslated strings now for you to edit

Reply:
Thanks!